src/components/UI/Display/NotAvailable/index.tsx

```diff
@@ -0,0 +1,25 @@
+// NotAvailable
+//
+// A placeholder label to be used where we have no data available to display.
+//
+
+import React, { FC } from 'react';
+import styled from 'styled-components';
+
+const NotAvailableOuter = styled.span`
+  opacity: 0.5;
+`;
+
+interface INotAvailableProps extends React.HTMLAttributes<HTMLDivElement> {}
+
+const NotAvailable: FC<INotAvailableProps> = ({ ...rest }) => {
+  return (
+    <NotAvailableOuter {...rest}>
```

Comment:
No need to use the rest operator twice

```suggestion
const NotAvailable: FC<INotAvailableProps> = (props) => {
  return (
    <NotAvailableOuter {...props}>
```